tools/apigee-sackmesser/cmd/report/report.sh

```diff
@@ -423,6 +423,16 @@ if [ "$opdk" == "T" ]; then
     source $opdk_dir/virtualhosts.sh
 fi
 
+if [ "$apiversion" = "google" ]; then
```

Comment:
Let's explore consolidation of conditionals in else if format to maintain code coherence.

Reply:
I'm with Anmol. If possible can we keep the code split between OPDK and x/hybrid as minimal as possible. 
In the past we tried to [harmonize](https://github.com/apigee/devrel/blob/main/tools/apigee-sackmesser/cmd/list/list.sh#L31-L51) the output format of `sackmesser list` and then use the same logic in report based on the unified data model.

Reply:
@g-devpat this was resolved without comment. 
I strongly believe we increase the  maintainability cost if we duplicate logic for X and OPDK.

Reply:
@danistrebel Yes, Agreed. earlier we were using seperate helper files for opdk and x which I eliminated as part of this commit 5100fc5. Now we have a single set of helper files for all Apigee versions